Snake/BusinessLayer/GameSnake.cs

```diff
@@ -80,6 +83,12 @@ public List<Point> snakeMove(int direction, Boolean hasEaten)
             return snakeMoveList;
         }
 
+        //return current snake position
+        public List<Point> returnCurrentSnakePosition()
```

Comment:
what does return mean here? is this just a Getter? if so just make it a get property and give it a noun for a name eg Position. 
Also since this just wraps a class variable you may as well just make that a public property